tools/manifest/sourcefile.py

```diff
@@ -865,6 +865,16 @@ def manifest_items(self):
                     self.rel_path
                 )]
 
+        elif self.name_is_webdriver:
```

Comment:
Oof, nice catch. Could we add a test so future-us don't get hurt when we refactor this?